__tests__/app.test.js

```diff
@@ -72,6 +72,40 @@ describe('GET /api', () => {
         })
     })
 })
+
+
+describe("GET /api/articles/:article_id", () => {
+    test("responds with a 200 status code", () => {
+        return request(app).get("/api/articles/3").expect(200)
+            .then((res) => {
+                console.log(res.body)
+                expect(res.body.article.article_id).toBe(3)
+                const numCreatedAt = new Date(res.body.article.created_at).getTime()
```

Comment:
As the reseed plays with this, you don't need to go this far to assert the key. I'd be more interested that the other keys are present and correct.